statistics/cmsketch.go

```diff
@@ -180,6 +181,21 @@ func (c *CMSketch) findTopNMeta(h1, h2 uint64, d []byte) *TopNMeta {
 	return nil
 }
 
+// MemoryUsage returns the total memory usage of a CMSketch in B.
+// We ignore the size of other metadata in CMSketch
+func (c *CMSketch) MemoryUsage() (sum int64) {
+	if c == nil {
+		return
+	}
+	sum = int64(cap(c.table) * 4)
+	for _, hs := range c.topN {
```

Comment:
1. It seems `cap(map)` can not get the memory usage of a map.
2. Can we ignore the memory usage of c.topN since it is small